lib/parser.ml

```diff
@@ -40,8 +40,39 @@ let expand_list_elem ~loc cont_expr = function
         | [%p some_pat ~loc pat] -> [%e junk ~loc cont_expr]
         | _ -> [%e raise_err_exn ~loc]]
 
+let bind_stream_in ~loc var_pat e2 =
+  [%expr
+    let [%p var_pat] = ppx____parser____stream____ in
+    [%e e2]]
+
+let error_stream_binding_end_of_pattern ~loc =
+  Err.err_expr_node ~loc "'%%stream' binding must come at the end of the pattern"
```

Comment:
Change this to: _"The '%%stream' binding can only be used at the end of the pattern."_?

Reply:
I've updated the wording, thanks